lib/alephant/renderer/views/html.rb

```diff
@@ -1,64 +1,57 @@
-require "alephant/renderer/views/base"
-require "mustache"
-require "i18n"
+require 'alephant/renderer/views/base'
+require 'alephant/renderer/engine/mustache'
+require 'alephant/renderer/i18n/locale_component_yaml'
 
 module Alephant
   module Renderer
     module Views
-      class Html < Mustache
+      class Html
         include ::Alephant::Renderer::Views::Base
 
+        class << self
+          # FIXME: remove this when we've gotten rid of all of the `self.template_path` directives
```

Comment:
That is covered here: https://github.com/BBC-News/alephant-renderer/pull/22/files/37a11ca6a5babd72b16e681f88ff8de1860f086b#diff-ca9488e2aa6446f753ce07b2829d39b1R12

Jodie moved it more to an application wide convention rather than having to specify it in each model.